Dockerfile

```diff
@@ -37,8 +37,14 @@ COPY root /
 # add container files
 COPY files /files
 
-RUN mkdir -p ${HOME}/src
+RUN touch /etc/service.conf && \
+    mkdir -p ${HOME}/src && \
+    useradd -u 1001 -r -g 0 -s /sbin/nologin service && \
+    /usr/libexec/fix-permissions /etc/service.conf ${HOME}/src/ && \
+    usermod -a -G root service
```

Comment:
Gotta say I don't understand this one: it's almost like a root user (I was able to mess around in `/usr/bin`, I couldn't change packages using nor dnf nor rpm though)